src/compiler/types.ts

```diff
@@ -822,7 +822,7 @@ namespace ts {
         expression: Expression;
     }
 
-    export interface PrivateName extends Declaration {
+    export interface PrivateName extends Expression, Declaration {

```

Comment:
Why is a `PrivateName` a `Declaration`? It seems to me like it would just be (one option for) the name of `NamedDeclaration`.

Reply:
good 👀 .
I made it an Expression to debug something else and forgot to switch it back.
Should just be `Declaration`. Needs to be a declaration because binder::addDeclarationToSymbol expects a declaration.